docs/specs/authentication-flow-api-reference.md

```diff
@@ -210,6 +231,14 @@ When you are in this step of this flow, you will see a response like the followi
     "state_token": "authflowstate_5R6NM7HGGKV64538R0QEGY9RQBDM4PZD",
     "type": "signup",
     "name": "default",
+    "captcha": {
+      "providers": [
+        {
+          "type": "cloudflare",
+          "alias": "cloudflare"
+        }
```

Comment:
Do we need to show extra information like `site_key` here? 🙏

Reply:
No